openrtb_ext/imp_adview.go

```diff
@@ -0,0 +1,6 @@
+package openrtb_ext
+
+type ExtImpAdview struct {
```

Comment:
It looks like your company name is "AdView". You are welcomed to rename `ExtImpAdview` to `ExtImpAdView` to better match your brand.

Reply:
Thx for your suggestion, we have changed it.